Framework/Geometry/inc/MantidGeometry/Objects/BeamlineRayTracer.h

```diff
@@ -0,0 +1,525 @@
+#ifndef MANTID_GEOMETRY_BEAMLINERAYTRACER_H_
```

Comment:
Why is this header-only? 🤔

Reply:
I think that might be a hangover from how Bhuvan started to write this. I can't see why we don't have a separate implementation file, and I think I discussed that with him.